generators/kubernetes/templates/README-KUBERNETES.md.ejs

```diff
@@ -162,6 +167,10 @@ Events:
 
 cert-manager first creates a temporary certificate. Once you see the event "The certificate has been successfully issued", it means Let's Encrypt staging has issued the certificate. The staging certificate will be trusted by deployed applications, but not by browsers. Let's Encrypt production certificates are trusted by browsers, but the production issuer will probably not work with nip.io, and fail with "too many certificates already issued for nip.io".
 
+<<<<<<< HEAD
+  <%_ } _%>
+=======
+>>>>>>> upstream/main
```

Comment:
It looks like a merge issue needs to be resolved here.

Reply:
Fixed